src/main/scala/com/databricks/spark/xml/XmlOptions.scala

```diff
@@ -64,6 +64,7 @@ private[xml] class XmlOptions(
     parameters.getOrElse("wildcardColName", XmlOptions.DEFAULT_WILDCARD_COL_NAME)
   val ignoreNamespace = parameters.get("ignoreNamespace").map(_.toBoolean).getOrElse(false)
   val timestampFormat = parameters.get("timestampFormat")
+  val timeZone = parameters.getOrElse("timeZone", XmlOptions.DEFAULT_TIME_ZONE)
```

Comment:
Instead of exposing an option, use the value of Spark conf `spark.sql.session.timeZone` below. We would need to document this behavior in the README